applications/GeoMechanicsApplication/tests/flow_rate_heterogeneous_soil.gid/ProjectParameters.json

```diff
@@ -32,7 +32,6 @@
         "block_builder":                      true,
         "solution_type":                      "Steady_State_Groundwater_Flow",
         "scheme_type":                        "Backward_Euler",
-        "reset_displacements":                true,
         "newmark_beta":                       0.25,
```

Comment:
Indeed it can be deleted, the defaults in geomechanics_U_Pw_solver.py and in geomechanics_solver.py are both false.
( In future we probably should make sure there is only 1 place where this setting has its default. )